contracts/RealmDiamond/facets/AlchemicaFacet.sol

```diff
@@ -221,6 +227,150 @@ contract AlchemicaFacet is Modifiers {
+
+  function _calculateSplit(
+    uint256 _amount,
+    uint256 _spilloverRate,
+    uint16 ownerShare,
+    uint16 borrowerShare,
+    address[][] memory recipients,
+    uint16[][] memory sharesArray,
+    address[] memory tokenAddresses,
+    uint256 _realmId
+  ) internal view returns (SplitCalculation memory splitCalc) {
+    (uint256 borrowerAmount, uint256 ownerAmount, uint256 remainingAmount) = _calculateAmounts(_amount, _spilloverRate, ownerShare, borrowerShare);
+
+    splitCalc.borrowerAmount = borrowerAmount;
+    splitCalc.ownerAmount = ownerAmount;
+    splitCalc.remainingAmount = remainingAmount;
+
+    (splitCalc.splitRecipients, splitCalc.recalculatedShares, splitCalc.splitAmounts, splitCalc.splitTokenAddresses) = _calculateTokenSplits(
+      TokenSplitParams({
+        remainingAmount: remainingAmount,
+        recipients: recipients,
+        sharesArray: sharesArray,
+        tokenAddresses: tokenAddresses,
+        ownerShare: ownerShare,
+        borrowerShare: borrowerShare
+      }),
+      _realmId
+    );
+  }
+
+  function _handleTokenChanneling(uint256 _realmId, bytes32 _roleId, uint256 channelAmount, uint256 tokenIndex) internal {
+    InstallationAppStorage storage si = LibAppStorageInstallation.diamondStorage();
+    (uint256 rate, ) = InstallationDiamondInterface(s.installationsDiamond).spilloverRateAndRadiusOfId(s.parcels[_realmId].altarId);
+    ProfitShare storage profitShare = s.profitShares[si.realmDiamond][_realmId][_roleId];
+    uint256 _gotchiId = _tempGotchiId;
+
+    IERC20Mintable alchemica = IERC20Mintable(profitShare.tokenAddresses[tokenIndex]);
+
+    if (alchemica.balanceOf(address(this)) < s.greatPortalCapacity[tokenIndex]) {
+      TransferAmounts memory amounts = calculateTransferAmounts(channelAmount, rate);
+
+      if (isLandRented(profitShare.tokenAddresses[tokenIndex], _realmId, _roleId)) {
+        _handleRentedLandChanneling(alchemica, _gotchiId, _realmId, channelAmount, rate, profitShare, tokenIndex);
+      } else {
+        alchemica.mint(LibAlchemica.alchemicaRecipient(_gotchiId), amounts.owner);
+        alchemica.mint(address(this), amounts.spill);
+      }
+    } else {
+      TransferAmounts memory amounts = calculateTransferAmounts(channelAmount, rate);
+      alchemica.transfer(LibAlchemica.alchemicaRecipient(_gotchiId), amounts.owner);
+    }
+  }
+
+  function _handleRentedLandChanneling(
+    IERC20Mintable alchemica,
+    uint256 _gotchiId,
+    uint256 _realmId,
+    uint256 channelAmount,
+    uint256 rate,
+    ProfitShare storage profitShare,
+    uint256 tokenIndex
+  ) internal {
+    SplitCalculation memory splitCalc = _calculateSplit(
+      channelAmount,
+      rate,
+      profitShare.ownerShare[tokenIndex],
+      profitShare.borrowerShare[tokenIndex],
+      profitShare.recipients,
+      profitShare.shares,
+      profitShare.tokenAddresses,
+      _realmId
+    );
+
+    alchemica.mint(LibAlchemica.alchemicaRecipient(_gotchiId), splitCalc.borrowerAmount);
```

Comment:
In this case just to you know, i didn't change this function. I kept because in the new Splitter we don't consider the borrowerAmount because this should be transferred out of split, as we talked first.